force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -161,38 +282,52 @@ public with sharing class FlowControlLoopsCollections {
      * @return The new string with the inputString repeated repeatCount times.
      */
     public static String repeatString(String inputString, Integer repeatCount) {
-        // Initialize the result String
 
+        // Initialize the result String
+        String repeatedString = '' ;
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
+        
+        for (Integer i = 1 ; i <= repeatCount ; i ++ ) {
+            // add inputString to the result string
+            repeatedString += inputString ;
```

Comment:
Dont leave the semicolon hanging out there by itself, should be part of your statement.